apps/camera/js/views/preview-gallery.js

```diff
@@ -27,7 +27,7 @@ require('gaia-header');
 var SWIPE_DISTANCE_THRESHOLD = window.innerWidth / 3; // pixels
 var SWIPE_VELOCITY_THRESHOLD = 1.0;                   // pixels/ms
 
-var SWIPE_DURATION = 250;   // How long to animate the swipe
+var SWIPE_DURATION = 150;   // How long to animate the swipe
```

Comment:
IMO 250ms swipe duration is long and reason grey screen is seen between images while swiping.